helix/index.rst

```diff
@@ -10,4 +10,5 @@ Sitecore Helix Documentation
 
     introduction/index
     principles/index
-    devops/index
\ No newline at end of file
+    devops/index
+    package-principles
```

Comment:
We should probably add a new Appendix section, where we can also add a page for "Additional Resources" -- blogs, videos, training, etc.

I can refactor to this after merge if needed.

Reply:
Did this, and built to HTML, and it looks pretty good:
![image](https://user-images.githubusercontent.com/689532/65821815-ba04c000-e208-11e9-9ec9-a55a658352f6.png)
![image](https://user-images.githubusercontent.com/689532/65821832-ff28f200-e208-11e9-919d-38b1a1b8d1c5.png)
![image](https://user-images.githubusercontent.com/689532/65821847-28e21900-e209-11e9-9282-c97ceda25e6f.png)